### ISSUE TITLE
If there is applied a custom SQL, after adding a widget, it doesn't respond to changes in the form

### ISSUE THREAD

**Issue by xavijam** [CartoDB/cartodb#6688]
### STR

- Create a map with populated places dataset.
- Go to old editor, and apply a custom SQL: 
```
SELECT *,( SELECT count(*) FROM table_name) as total_amount FROM table_name
```
- Go to the new editor url.
- Add a new formula widget with `total_amount` column.
- It will appear correctly (yay).
- Go to widgets list and select the previous widget, let's edit it.
- Curious thing, `total_amount` doesn't appear as chosen column :|.
- Try to change the value attribute to another value, it doesn't change :(.

**Comment by xavijam** [comment]
is it already done right?

**Comment by viddo** [comment]
yes (verified working), the error at the time you wrote the issue was probably the same that was solved by #6741
